### ISSUE TITLE
Force save to go on SD

### ISSUE THREAD

**Issue by LukasAppleFan** [FIX94/Nintendont#895]
Here is my original post on Reddit :
Hello,

I am on a Wii U but this is vWii related so not sure if this sould be posted here but anyway.

I use the Wii U to play GameCube games, and I launch games from eithe the Wii U or the vWii menu.

For the Wii U menus I use VC injects so the games are the HDD that is Wii U formatted and the saves are on the sd:/saves, but when I load games from the USB that I use with USB Loader GX, then the saves are in usb:/saves.

And I wanted to know if I could put all games save to one location (I would prefer SD).

I didn't find any option in Nintendont, or USB Loader GX to change the save path, only option I found was that emu save thing but that isn't related to my issue.

Thanks in advance.

TL;DR I would to have an option to only use the SD to store saves

**Comment by LukasAppleFan** [comment]
Ok thanks for your reply, but just to clarify when you say : 1-nintendont cant mount both usb and sd at the same time only one.
Does it mean that the way the Wii (U)/vWii is made it's impossible to make it work. Or will someone find a way ?

**Comment by carnage702** [comment]
> 
> 
> Ok thanks for your reply, but just to clarify when you say : 1-nintendont cant mount both usb and sd at the same time only one.
> Does it mean that the way the Wii (U)/vWii is made it's impossible to make it work. Or will someone find a way ?

its simply a memory limitation, nintendont cant run and mount 2 different devices being it on wii or wiiu makes no difference since the memory available is small.

**Comment by LukasAppleFan** [comment]
Ok thanks for the details
